tools/pre-commit.sh

```diff
@@ -1,28 +1,83 @@
 #!/usr/bin/env bash
+# Copy or link this script into .git/hooks/
+#   $ cp "`pwd`/tools/pre-commit.sh" "`pwd`/.git/hooks/pre-commit"
+# This script runs automatically in the project's root directory (parent of .git/).
 
-python -m black --check ./ > /dev/null 2>&1
-need_format=$?
-set -e
-if [ $need_format -ne 0 ]
-then
-    python -m black ./
-    echo Some Python files were formatted
-    echo You need to do git add and git commit again
-    exit $need_format
+EXIT_STATUS=0
+EXIT_EARLY=0
+MAX_FILE_SIZE=32 # Max JSON file size in Megabytes.
```

Comment:
Is this standard max size for bash variables?

Reply:
This is up to the developer.